### ISSUE TITLE
`ClosureAsLastMethodParameter` vs. a vararg list of closures

### ISSUE THREAD

**Issue by rkrisztian** [CodeNarc/CodeNarc#629]
I've got a test helper method like this:

```
class GroovyAssertions {

	/**
	 * See <a href="https://issues.apache.org/jira/browse/GROOVY-8045">GROOVY-8045</a>.
	 */
	static void assertAll(Closure... executables) {
		Assertions.assertAll executables.collect { it as Executable }
	}

}
```

And I use it like this:

```
		assertAll(
				{ assert a == b },
				{ assert list.size() == 2 },
				{ assert !someOthercondition }
		)
```

However, rule `ClosureAsLastMethodParameter` wants to rewrite the last parameter in this case too, which I don't think would be more readable.

**Comment by chrismair** [comment]
I am considering adding an `ignoreMethodNames` property to that rule, since I do not have a good solution otherwise.

**Comment by rkrisztian** [comment]
That would work for me, thanks.

**Comment by rkrisztian** [comment]
To be honest, I'm considering the switch to Spock, however, the issue might stay valid for other exceptional use cases...

**Comment by chrismair** [comment]
Added the `ignoreCallsToMethodNames` property to that rule. Will be available with CodeNarc 2.2.0.
